orchestration/utils/config.py

```diff
@@ -49,7 +53,68 @@ def init_logging():
     logger.addHandler(server_log_file)
 
 
+def init_config(file='orchestration.conf'):
+    global conf
+    global HOST
+    global PORT
+
+    conf = configparser.ConfigParser()
+    dataset = conf.read(file)
+    if len(dataset) == 0:
+        logger.error("Failed to open orchestration config file: [%s]" % file)
+        file = 'orchestration.conf'
+        logger.warning("Creating default config file at: [%s]" % file)
+        # create default template config file
+        cfgfile = open(file, 'w')
+
+        # fill config file with default data
+        conf.add_section('orchestration')
+        conf.set('orchestration', 'host', HOST)
+        conf.set('orchestration', 'port', PORT)
+        conf.add_section('workflow')
+        conf.set('workflow', 'tech', 'St2')
+        conf.set('workflow', 'host', '127.0.0.1')
+        conf.set('workflow', 'username', 'St2')
+        conf.set('workflow', 'encripted_password', 'false')
+        conf.set('workflow', 'password', 'password')
+        conf.set('workflow', 'phrase', '')
+
+        conf.write(cfgfile)
+        cfgfile.close()
+
+    HOST = conf['orchestration']['host']
+    PORT = conf['orchestration']['port']
+
+
+def get_workflow_config():
+    global conf
+    tech = ''
+    server = ''
+    user = ''
+    passwd = ''
+
+    try:
+        server = conf.get('workflow', 'host')
+        user = conf.get('workflow', 'username')
+        tech = conf.get('workflow', 'tech')
+        encripted = conf.get('workflow', 'encripted_password')
+        if encripted == 'false':
+            passwd = conf.get('workflow', 'password')
+        else:
+            phrase = conf.get('workflow', 'phrase')
+            ciphered_suite = Fernet(phrase.encode())
+            passwd = (ciphered_suite.decrypt(passwd.encode()))
+    except Exception as ex:
+        print(passwd)
```

Comment:
remove this as if the passwd is not encrypted, we may print it on console. Also, can we use logger module to log this than to print on stdout

Reply:
Done